tests/pool_test.go

```diff
@@ -116,11 +117,24 @@ var _ = Describe("[sig-compute]VirtualMachinePool", decorators.SigCompute, func(
 	newPersistentStorageVirtualMachinePool := func() *poolv1.VirtualMachinePool {
 		By("Create a new VirtualMachinePool with persistent storage")
 
-		vm, foundSC := tests.NewRandomVMWithDataVolume(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros), util.NamespaceTestDefault)
-		if !foundSC {
+		sc, exists := libstorage.GetRWOFileSystemStorageClass()
+		if !exists {
 			Skip("Skip test when Filesystem storage is not present")
 		}
 
+		dataVolume := libdv.NewDataVolume(
+			libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros)),
+			libdv.WithPVC(libdv.PVCWithStorageClass(sc)),
+		)
+
+		vm := libvmi.NewVirtualMachine(
+			libvmi.New(
+				libvmi.WithDataVolume("disk0", dataVolume.Name),
+				libvmi.WithResourceMemory("100M"),
```

Comment:
Do we test anything about the guest? Does it even boot? You may waste even less memory https://github.com/kubevirt/kubevirt/pull/11033/files#diff-005747350099de38fcd22401693fe634b2c4607b4deec581b298efe440b93d2fL183